### PR TITLE
Restructure

### LINKED ISSUE #5: Update Synths SDK

### LINKED ISSUE #20: SDK: Add contracts and hardhat config

### PR DESCRIPTION
**Pull request by ghost**
### Description of changes proposed in this pull request:
- Restructure sdk
- Add synth test

### Checks:
* [x] Have you followed the guidelines in our Contributing document?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### CODE REVIEW

File: src/types/assets.t.ts [hunk 1]
```diff
@@ -1,61 +1,89 @@
...
+  collateral: string;
+  /** token - Token of the asset */
+  token: TokenConfig;
+  /** emp - Emp of the asset */
+  emp: EmpConfig;
+  /** pool - Pool of the asset */
+  pool: PoolConfig;
+  /** expired - Identifier for the expiry of the asset */
+  expired: boolean;
 }
```
Comment (ethedev) [inline src/types/assets.t.ts:79]: to keep comments where its really needed, instructive and not all over the place
- eventually everywhere and later after this pr, we will need to add it on the top of the object like that:
```js
/**
 * Asset specifications
 *
 * @member {string} name
 * @member {string} cycle
 * @member {string} year
 * @member {string} collateral
 * @member {TokenConfig} token
 * @member {EmpConfig} emp
 * @member {PoolConfig} pool
 * @member {boolean} expired Force expiration of the asset
 */
export interface AssetConfig {
```

```suggestion
/** AssetConfig - Asset specifications */
export interface AssetConfig {
  name: string;
  cycle: string;
  year: string;
  collateral: string;
  token: TokenConfig;
  emp: EmpConfig;
  pool: PoolConfig;
  expired: boolean; // Force expiration of the asset
}
```

File: test/Synths.test.ts [hunk 2]
```diff
@@ -0,0 +1,187 @@
+import { ethers, waffle } from "hardhat";
```
Comment (ethedev) [inline test/Synths.test.ts:1]: what is the difference `Synths` and `SynthsSdk` files? i can think of Synths as the Sdk and the opposite, the naming on these should be more accurate

**Review by ethedev — CHANGES_REQUESTED** [review]

File: test/Synths.test.ts (hunk 2 above)
Comment (byterose) [reply test/Synths.test.ts:1]: ```Synths.test``` tests a lspfpl synth. What do you think about ``Àsset.test``` instead?
Or should we name it ```LSPFPL.test```?

Comment (ethedev) [reply test/Synths.test.ts:1]: good to separate them each into its own file
- `EMP-Synths.test`
- `FPL-Synths.test`
- `LSP-Synths.test`

**Review by ethedev — APPROVED** [review]

**Pull request merged**